src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -557,6 +559,7 @@ public static int TrailingZeroCount(int value)
         /// Similar in behavior to the x86 instruction TZCNT.
         /// </summary>
         /// <param name="value">The value.</param>
+        [Intrinsic]
         [CLSCompliant(false)]
```

Comment:
It is not needed for mono runtime. I think it is good to use it though, so that it is clear when reading the C# code. We do the same for S.R.I.Wasm.PackedSimd and WasmBase classes.

Reply:
Are these intrinsics for coreclr ? It looks like they just contain calls to intrinsics.